### ISSUE TITLE
ability to control message formatting to slack

### ISSUE THREAD

**Issue by dszabo** [ontometrics/slack-youtrack#2]
It'd be a nice feature if I was able to control how the messages are sent to slack with slack formatting (for example, the message content should be sent between three and three backticks in order to display as a code block on the slack channel) like this:

```
would be nice to display the posted content as a code block
```

**Comment by codeslubber** [comment]
Do they support that now? I thought when I looked at their API while writing this that was not supported. Will look again.

**Comment by codeslubber** [comment]
Just looked and code blocks are supported.

**Comment by dszabo** [comment]
yes, they do. also, I noticed that you replace curly braces to [ and ]. Any particular reason for that? I am guessing that otherwise it causes issues to slack's json data?
